L1Trigger/L1TTrackMatch/plugins/L1TrackTripletEmulatorProducer.cc

```diff
@@ -277,14 +277,13 @@ void L1TrackTripletEmulatorProducer::produce(Event &iEvent, const EventSetup &iS
 
   if (!event_pass) {
     iEvent.put(std::move(L1TrackTripletContainer), OutputDigisName);
+    iEvent.put(std::move(L1TrackTripletWordContainer), OutputWordName);
```

Comment:
what is the difference between L279 and L280?

Reply:
The 279 returns an empty container of the triplet track, while 280 of the triplet track eord. The first is the cpp object with fll pointers for tracks meant for checking/improving the code, while the latter is similar to the output of the firmware